internal/runner/accept.go

```diff
@@ -38,8 +39,14 @@ func (r *Runner) AcceptExact(ctx context.Context, id string) error {
 	return r.accept(ctx, id)
 }
 
+var testRecvDelay time.Duration
+
 func (r *Runner) accept(ctx context.Context, id string) error {
-	if r.closed() {
+	r.runningMu.Lock()
```

Comment:
If this is just equivalent to `runnerCond.L` anyways, I think it's better to just use the locker on the cond var and not store `r.runnerMu` at all.

Reply:
Ah! I started that but didn’t finish it. Gotcha.